test/app/lib/CoursesHelper.spec.coffee

```diff
@@ -0,0 +1,115 @@
+helper = require 'lib/coursesHelper'
+Campaigns = require 'collections/Campaigns'
+Users = require 'collections/Users'
+Courses = require 'collections/Courses'
+CourseInstances = require 'collections/CourseInstances'
+Classrooms = require 'collections/Classrooms'
+
+describe 'CoursesHelper', ->
+
+  describe 'calculateAllProgress', ->
+
+    beforeEach ->
+      # classrooms, courses, campaigns, courseInstances, students
+      @classrooms = require 'test/app/fixtures/classrooms'
+      @classroom = @classrooms.models[0]
+      @courses = require 'test/app/fixtures/courses'
+      @course = @courses.models[0]
+      @campaigns = require 'test/app/fixtures/campaigns'
+      @campaign = @campaigns.models[0]
+      @students = require 'test/app/fixtures/students'
+
+    describe 'when all students have completed a course', ->
+
+      beforeEach ->
+        @classroom.sessions = require 'test/app/fixtures/level-sessions-completed'
+        @courseInstances = require 'test/app/fixtures/course-instances'
+
+      it 'progressData.get({classroom, course}) returns object with .completed=true and .started=true', ->
```

Comment:
The thing being tested should be in the `describe`, behavior in `it`. So

```coffeescript
describe 'progressData.get({classroom, course})', ->
  describe 'when all students have completed a course', ->
    it 'returns an object with .completed=true and .started=true'
  describe 'when something is a certain way', ->
    it 'returns a thing'
```

That way you can read the `describe` and `it` lines and they're grammatical.